client/export.go

```diff
@@ -0,0 +1,312 @@
+// Copyright 2025 SAP SE
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"slices"
+	"time"
+
+	"github.com/cheggaaa/pb/v3"
+	"github.com/sapcc/go-bits/logg"
+	"github.com/sapcc/gophercloud-sapcc/v2/audit/v1/events"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v3"
+)
+
+type ExportFormat string
+
+const (
+	ExportFormatJSON ExportFormat = "json"
+	ExportFormatYAML ExportFormat = "yaml"
+	ExportFormatCSV  ExportFormat = "csv"
+)
+
+var (
+	allExportFormats = []ExportFormat{ExportFormatJSON, ExportFormatYAML, ExportFormatCSV}
```

Comment:
I'd split them into multiple lines. It would be more readable for further changes.

Reply:
Thanks, appreciate you.